Aptoide/aptoide-client-v8
# [task/ASV-2061] Apps refactor

**What does this PR do?**

   This PR includes ASV-1895, ASV-2061 and ASV-2062. It refactors the Apps view to use Epoxy and a custom view that represents the download state / progress. Note that because of major changes, SeeMoreAppcFragment is currently removed (only 5 appc migrations will be shown at a time). 

**Database changed?**

   No

**How should this be manually tested?**

  Check that every flow works the same as the previous version of Apps (the current version also has some bugs), e.g. Update all, download/updates (and their controls).

**What are the relevant tickets?**

  [ASV-1895](https://aptoide.atlassian.net/browse/ASV-2061)
  [ASV-2061](https://aptoide.atlassian.net/browse/ASV-2061)
  [ASV-2062](https://aptoide.atlassian.net/browse/ASV-2062)

**Code Review Checklist**
